src/guide/data_types.md

```diff
@@ -151,3 +151,32 @@ print({ name: 'Klaus Teuber', age: 42 })
 ```
 
 We haven't discussed exports yet, but we'll go much deeper into them in another section.
+
+## Mutable record properties
+
+We've previously creating mutable `let` bindings, but you can also label individual properties on your records `mut`.
+
+```grain
+export data Counter = {
+  mut count: Number
+}
+
+let counter = { count: 0 }
+
+let incr = (c) => {
+  c.count = c.count + 1
+}
+
+let decr = (c) => {
+  c.count = c.count - 1
+}
```

Comment:
I like this example, though I'm not sure it makes sense in the docs since `incr` and `decr` both shadow real values in Pervasives.

Reply:
I'll change to `addOne` and `subtractOne` and then use the `incr` and `decr` from Pervasives. I like that more.